invenio_vocabularies/alembic/1d3302223885_create_vocabularies_subtype.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 Northwestern University.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""create_vocabularies_subtype."""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '1d3302223885'
+down_revision = '4f365fced43f'
```

Comment:
Not ultra familiar with alembic, I guess it supports branching. i.e. both this and the affiliations recipe have the same `down_version`. Just trying to make sure none will be skipped.

Reply:
They won't be skipped but the next migration will cause a slight headache for the migrator. They could merge them (https://alembic.sqlalchemy.org/en/latest/branches.html#merging-branches), but we might as well make the migrations follow one another and simplify everybody's life. I will "rebase" the migration on top of the latest one (affiliations one).

Reply:
Thanks for the reference, reading it now :)